internal-webapps/org.wso2.bfsi.authentication.endpoint/src/main/webapp/bfsi_default.jsp

```diff
@@ -0,0 +1,200 @@
+                        </c:when>
+                        <c:when test="${consent_type eq 'fundsconfirmations'}">
+                            <strong>${app}</strong> requests access to confirm the availability of funds in your account.
+                        </c:when>
+                        <c:when test="${consent_type eq 'payments'}">
+                            <strong>${app}</strong> requests consent to do a payment transaction ${intentSubText}
+                        </c:when>
+                    </c:choose>
+                </h3>
+
+                <h4 class="section-heading-5 ui subheading">Data requested:</h4>
+                <!--Display requested data-->
+                <c:forEach items="<%=consentData%>" var="record">
+                    <div class="padding" style="border:1px solid #555;">
+                        <b>${record.key}</b>
+                        <ul class="scopes-list padding">
+                            <c:forEach items="${record.value}" var="record_data">
+                                <li>${record_data}</li>
+                            </c:forEach>
+                        </ul>
+                    </div>
+                </c:forEach>
+            </div>
+
+            <%-- Setting data based on the consent type --%>
+            <c:choose>
+                <%-- Setting accounts related data --%>
+                <c:when test="${consent_type eq 'accounts'}">
+                    <div class="form-group ui form select">
+                        <c:if test="${not empty account_data}">
+                            <h5 class="ui body col-md-12">
+                                Select the accounts you wish to authorise:
+                            </h5>
+                            <div class="col-md-12" >
+                                <c:forEach items="${account_data}" var="record">
+                                    <label for="${record['display_name']}">
+                                        <input type="checkbox" id="${record['display_name']}" name="chkAccounts"
+                                               value="${record['account_id']}" onclick="updateAcc()"
+                                        />
+                                            ${record['display_name']}
+                                    </label>
+                                    <br>
+                                </c:forEach>
+                            </div>
+                        </c:if>
+                    </div>
+                </c:when>
+
+                <%-- Setting confirmation of funds related data --%>
+                <c:when test="${consent_type eq 'fundsconfirmations'}">
+                    <c:if test="${isReauthorization eq true}">
+                        <div class="form-group ui form">
+                            <strong>
+                                Note: You are about to re-authorize an intent which had been authorized before.<br/>
```

Comment:
We have used both "authorize" and "authorise" in this page. Shall we stick to one?

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/8/commits/7c13892e0735c7d32a51565d4369ab1e9eddbc7e